emily/lambda/src/main.rs

```diff
@@ -0,0 +1,175 @@
+use aws_lambda_events::apigw::{ApiGatewayProxyRequest, ApiGatewayProxyResponse};
+use aws_lambda_events::encodings::Body;
+use emily::models::{CreateDepositRequestContent, DepositParameters};
+use http::{header, HeaderValue};
+use lambda_runtime::{service_fn, Error, LambdaEvent};
```

Comment:
Nit: I prefer not importing ubiquitous names such as `Error` without any additional namespacing as I find it reduces readability. I'd rather refer to it as `lambda_runtime::Error` in the code, or type alias it as

```suggestion
use lambda_runtime::{service_fn, Error as LambdaRuntimeError, LambdaEvent};
```

Reply:
I didn't know you could do that!

Reply:
Fixed.